README.md

```diff
@@ -16,6 +16,10 @@ It includes Docker Compose configuration to run the application stack locally.
 
 <div style="clear: both;">&nbsp;</div>
 
+**🚧 NOTE**:
+Because this product was initially made as part of a hackathon under very tight time and resource limitations, its architecture had to be rapidly improvised, and as such is a bit "rough around the edges".
+Our experienced team of engineers intends to turn this brainstorm/proof-of-concept into a fully-fledged, robust system through ongoing funded work.
```

Comment:
I feel we can accurately say this is a "proof" as it materialized outside our brains.
```suggestion
Our experienced team of engineers intends to turn this proof-of-concept into a fully-fledged, robust system through ongoing funded work.
```